### DISCUSSION TITLE
Graph not detecting Folders created in Browser

### DISCUSSION THREAD

**Discussion by dadams39** [microsoftgraph/msgraph-sdk-go#230] [Q&A]
The current documentation for [Hidden Folders](https://docs.microsoft.com/en-us/graph/api/user-list-mailfolders?view=graph-rest-1.0&tabs=go) states the following implementation:
```go
// Note: graphconfig "github.com/microsoftgraph/msgraph-sdk-go/users/item/mailfolders"
requestParameters := &graphconfig.MailFoldersRequestBuilderGetQueryParameters{
	IncludeHiddenFolders: true,
}
configuration := &graphconfig.MailFoldersRequestBuilderGetRequestConfiguration{
	QueryParameters: requestParameters,
}

result, err := graphClient.Me().MailFolders().GetWithRequestConfigurationAndResponseHandler(configuration, nil)
```

However, the request parameters no longer have this field. I bring this up because I have created folders in the `outlook.com` and I cannot see them when I run the last query.

**Comment by baywet** [comment]
Thanks for bringing this up.
This property has never been present in the code as far as I can tell because the query parameter is missing from the metadata, [I've created an issue so we can patch the metadata](https://github.com/microsoftgraph/msgraph-metadata/issues/196) like we've done for calendar view.

In the meantime, hardcoding the url by newing the request builder should help you to workaround the issue.
